CopilotKit/packages/react-ui/package.json

```diff
@@ -45,28 +45,10 @@
   "dependencies": {
     "@copilotkit/shared": "workspace:*",
     "@copilotkit/react-core": "workspace:*",
-    "@heroicons/react": "^2.0.18",
-    "@radix-ui/react-alert-dialog": "^1.0.4",
-    "@radix-ui/react-dialog": "^1.0.4",
-    "@radix-ui/react-dropdown-menu": "^2.0.5",
-    "@radix-ui/react-label": "^2.0.2",
-    "@radix-ui/react-select": "^1.2.2",
-    "@radix-ui/react-separator": "^1.0.3",
-    "@radix-ui/react-slot": "^1.0.2",
-    "@radix-ui/react-switch": "^1.0.3",
-    "@radix-ui/react-tooltip": "^1.0.6",
-    "class-variance-authority": "^0.6.1",
-    "clsx": "^1.2.1",
```

Comment:
`clsx` is pretty useful for constructing conditional CSS strings.
Right now we are doing this manually in a bunch of places, I saw it in a few places but neglected to mention it.

But i'm ok with deleting it here in this PR, and bringing it back separately when the relevant code arises again.

Reply:
We bring it back when needed 👍